front/src/applications/stdcmV2/components/StdcmResults/StdcmResults.tsx

```diff
@@ -130,6 +262,7 @@ const StcdmResults = ({
             isReadOnly
             hideAttribution
             showStdcmAssets
+            isNonFeasible={conflictData.status === 'conflicts'}
```

Comment:
Personally, I find logic easier to understand when there is no negation

```suggestion
            isFeasible={conflictData.status !== 'conflicts'}
```

Reply:
Fixed :)